apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/AbstractSamhsaMatcher.java

```diff
@@ -74,29 +69,31 @@ protected AbstractSamhsaMatcher() {
         resourceCsvColumnToList("samhsa-related-codes/codes-cpt.csv", "CPT Code").stream()
             .map(AbstractSamhsaMatcher::normalizeHcpcsCode)
             .collect(Collectors.toUnmodifiableSet());
-    this.icd9ProcedureCodes =
+    Set<String> procdureCodes =
```

Comment:
Concatenating both ICD 9 and 10 codes together here. This is a dumb but simple fix to ensure all codes are getting checked every time in case the code system is set incorrectly.